drv.c

```diff
@@ -833,3 +833,15 @@ uint32_t drv_get_max_texture_2d_size(struct driver *drv)
 
 	return UINT32_MAX;
 }
+
+bool drv_query_dev_feature(struct driver * drv)
```

Comment:
the function name seems incorrect, you are checking if blob feature supported in virtio-gpu driver.